python_ta/reporters/templates/template.txt

```diff
@@ -94,6 +107,12 @@
 		</a>
 	</div>
 </body>
+<script
```

Comment:
Now we will download and place this minified jquery in pyta/python_ta/reporters/templates/ and include it, 
similar to the include in the link below, but with different enclosing element, for inline scripts. look up how to do this.

Similarly, directly after the above script element, we can "include" the script.js -- we may be to put the includes in the same enclosing script element, otherwise in a new one directly after.

https://github.com/pyta-uoft/pyta/blob/master/python_ta/reporters/templates/template.txt#L4